modules/cli/src/main/scala/scala/cli/commands/Doctor.scala

```diff
@@ -67,10 +67,15 @@ object Doctor extends ScalaCommand[DoctorOptions] {
       println(
         s"scala-cli would not be able to update itself since it is installed in multiple directories: ${scalaCliPaths.mkString(", ")}."
       )
-    else
+    else if (Update.isScalaCLIInstalledByInstallationScript())
       println(
         s"scala-cli could update itself since it is correctly installed in only one location: $scalaCliPaths."
       )
+    else
+      println(
+        s"scala-cli could update by your package manager since it is correctly installed in only one location: $scalaCliPaths."
```

Comment:
NIT Phrasing, and don't leave `Seq(…)` or `List(…)` in the output.
```suggestion
        s"scala-cli can be updated by your package manager since it is correctly installed in only one location: ${scalaCliPaths.mkString}."
```

Reply:
Ideally, we should implement https://github.com/VirtusLab/scala-cli/issues/670#issuecomment-1123664191, and rely on it to know whether the `scala-cli` binary has been installed by a package manager.